src/components/Inputs/TextMaskedInput/TextMaskedInput.js

```diff
@@ -93,7 +93,7 @@ export const TextMaskedInput = (props) => {
   }
 
   const getClasses = () => {
-    return !!getError(currentError, whichTouched)
```

Comment:
Can we just configure eslint to allow this? I think it's a bit nuanced when this is good or bad.

Reply:
This seems more problematic. I.e., if you were expecting a boolean later and now you don't have one. Lemme see how to disable this.